pkg/engine/artifacts.go

```diff
@@ -21,6 +21,12 @@ type kubernetesFeatureSetting struct {
 
 func kubernetesContainerAddonSettingsInit(profile *api.Properties) map[string]kubernetesFeatureSetting {
 	return map[string]kubernetesFeatureSetting{
+		DefaultHeapsterAddonName: {
+			"kubernetesmasteraddons-heapster-deployment.yaml",
+			"kube-heapster-deployment.yaml",
+			!common.IsKubernetesVersionGe(profile.OrchestratorProfile.OrchestratorVersion, "1.12.0"),
```

Comment:
let's change this to 1.13.0, so that we don't change the presence of heapster in pre-existing 1.12 clusters

Reply:
Done